scripts/add_license_headers.py

```diff
@@ -1,196 +0,0 @@
-#!/usr/bin/env python
-# Copyright 2018-2022 Streamlit Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Updates the license on all our source files."""
-import sys
-import os.path
-import glob
-from datetime import datetime
-
-
-if sys.version_info < (3, 6):
-    print("This script must be run with Python >= 3.6")
-    sys.exit(-1)
-
-
-FOLDERS = [os.path.abspath(os.path.normpath(x)) for x in sys.argv[1:]]
-
-# TODO: Take these are input arguments.
-EXCLUDE_PATTERNS = [
-    "/autogen/",
-    "/bin/",
-    "/build/",
-    "/gen/",
-    "/static/",
-    "/vendor/",
-    "/node_modules/",
-    "/cypress/",
-    "/react-app-env.d.ts",
-    "/css/variables.scss",  # scss-to-json doesn't like our normal header.
```

Comment:
Do we need to ignore this file as well with the new version?

Reply:
I am not sure if this is needed but for the sake of safety I have excluded this file and then we can use a different style of comments as it is done now.
https://github.com/streamlit/streamlit/blob/9b173fef34e1c7b9347023a2e1eb6ee0abcf994c/frontend/src/assets/css/variables.scss#L15-L16